### ISSUE TITLE
elim use of CMP0054 OLD

### ISSUE THREAD

**Issue by cyrush** [LLNL/conduit#487]
Stop this policy choice:

```
# cmake var deref policy
 if(POLICY CMP0054)
   cmake_policy(SET CMP0054 OLD)
 endif()
```

@white238  explored this, and know if/where we actually rely on it, I think it was in only one place?
@white238 can you refresh my memory and we can resolve for 0.5.1

**Comment by cyrush** [comment]
https://stackoverflow.com/questions/45900159/how-to-use-variables-and-avoid-cmp0054-policy-violations

**Comment by cyrush** [comment]
resolved in #491
